nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -230,6 +230,7 @@ def compression_rate(self, compression_rate: float) -> None:
 
     def disable_scheduler(self):
         self._scheduler = StubCompressionScheduler()
+        self._scheduler.target_level = 0.0
```

Comment:
Looks like a crutch for `compression_stage` function. Would you suggest more general solution?

Reply:
Yes, it is. It bugs me too. I have created a ticket to fix it properly some time ago: 96828. In short, in my opinion the whole approach with `StubCompressionScheduler` is a hack in some way. I think every scheduler should have the disabling/enabling functionality. Then, we won't need to replace scheduler within controller, and will be able to disable one already present.